src/main/java/com/watch/shop/app/controller/InputHandler.java

```diff
@@ -0,0 +1,12 @@
+package com.watch.shop.app.controller;
+
+import java.util.Scanner;
+
+public class InputHandler {
```

Comment:
Scanner and InputHandler is part of User interaction, it's View component.

Please check MVC responsibliity by layer:

For example, we change HOW we connect with User, from console to file.
In your current state, there will be changes to Controller layer.
![1_y8Z4MgBS_s8d4o26arDJ4w](https://github.com/user-attachments/assets/26e687c8-109f-4f7c-8250-646e50586316)

But it should be View.